src/script/lua_api/l_internal.h

```diff
@@ -30,6 +30,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "common/c_internal.h"
 
 #define luamethod(class, name) {#name, class::l_##name}
+#define luamethod_bc(class, name, alias) {#name, class::l_##name}, {#alias, class::l_##name}
```

Comment:
backwards compatible.

I'm unsure about this name. Feel free to suggest another one, ideally 9 characters long to match `luamethod`